### PR TITLE
MF-588: Convert form renderer to carbon

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

File: projects/ngx-formentry/src/components/file-upload/secure.pipe.ts [hunk 1]
```diff
@@ -5,11 +5,7 @@ import {
...
 } from '@angular/core';
-import { of } from 'rxjs';
-
-import { Subscription } from 'rxjs';
-import { Observable } from 'rxjs';
-import { BehaviorSubject } from 'rxjs';
+import { of ,  Subscription ,  Observable ,  BehaviorSubject } from 'rxjs';
 
 import { DomSanitizer } from '@angular/platform-browser';
 // import { FileUploadResourceService } from '../../etl-api/file-upload-resource.service';
```
Comment (drizzentic) [inline projects/ngx-formentry/src/components/file-upload/secure.pipe.ts:11]: Do we anticipate to use this import anywhere?

File: projects/ngx-formentry/src/components/ngx-date-time-picker/ngx-date-time-picker.component.ts [hunk 2]
```diff
@@ -56,13 +61,25 @@ export class NgxDateTimePickerComponent
...
-  public onTouched: any = () => {};
+  public onTouched: any = () => {
+
+  };
   public ngOnInit() {}
 
   public writeValue(value) {
-    this.value = value;
+    if(value instanceof Date){
+      console.log('Date', value)
```
Comment (drizzentic) [inline projects/ngx-formentry/src/components/ngx-date-time-picker/ngx-date-time-picker.component.ts:74]: We should consider removing debug statements.

File: projects/ngx-formentry/src/components/ngx-datetime-picker/ngx-datetime-picker.component.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  value: String | Date = '';
+  isDisabled: boolean;
+
+  onChange = (_: any) => { }
+  onTouch = () => { }
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```
Comment (drizzentic) [inline projects/ngx-formentry/src/components/ngx-datetime-picker/ngx-datetime-picker.component.ts:28]: Is this intentional or should we consider removing empty constructor and init functions?

File: projects/ngx-formentry/src/components/ngx-remote-select/remote-select.component.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    (deselected)="removed($event)"
+    [options]="items"
+    [allowClear]="true"
+    [placeholder]="placeholder"
+    [notFoundMsg]="notFoundMsg"
+    (typed)="typed($event)"
+    tabindex="0"
+  >
+  </ng-select>
+  <div *ngIf="loading">resolving....</div> -->
```
Comment (drizzentic) [inline projects/ngx-formentry/src/components/ngx-remote-select/remote-select.component.html:15]: Any reason why we should maintain this?

File: projects/ngx-formentry/src/components/ngx-date-time-picker/ngx-date-time-picker.component.html [hunk 5]
```diff
@@ -1,53 +1,34 @@
...
-        *ngIf="showTimePicker"
-        [initTime]="value"
-        [use12Hour]="true"
-        (onSelectTime)="onTimeSelect($event)"
-        (onTimePickerCancel)="toggleTimePicker($event)"
-      ></time-picker>
+    <div class="col">
+      <div class="bottom_aligner"></div>
+      <div  *ngIf="showTime">
+        <!-- <ngx-time-picker  [initTime]="value" [use12Hour]="true"
```
[the remark this answers is not in the document]
Comment (enyachoke) [reply projects/ngx-formentry/src/components/ngx-date-time-picker/ngx-date-time-picker.component.html:21]: I will remove the component in favor a new one.

File: projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.ts [hunk 6]
```diff
@@ -3,19 +3,25 @@ import {
   OnInit,
   Input,
   Inject,
+  ViewChild,
+  QueryList,
   OnChanges,
   SimpleChanges
 } from '@angular/core';
 import 'hammerjs';
+import { Tabs, Tab } from 'carbon-components-angular';
```
[the remark this answers is not in the document]
Comment (enyachoke) [reply projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.ts:12]: Actually both of them will be removed eventually but in the meantime I will prefer the angular tabs.

**Review by drizzentic — APPROVED** [review]
LGTM!

**Pull request merged**